### ISSUE TITLE
prefect.io UI service

### ISSUE THREAD

**Issue by lmeyerov** [TheDataRideAlongs/ProjectDomino#18]

**Comment by ZiyaoWei** [comment]
After #37 and #40 this is mostly done - might still want separate CPU and GPU agents. I can open a new issue for that if you think it's not urgent @lmeyerov. FWIW that should be easy to implement using [Affinity](https://docs.prefect.io/orchestration/agents/overview.html#flow-affinity-labels).
